docs/deployment/developing_in_a_GitHub_Codespaces.md

```diff
@@ -1,27 +1,27 @@
 # Developing in a GitHub Codespaces
 
 - [Developing in a GitHub Codespaces](#developing-in-a-github-codespaces)
-  - [Getting started with codespace](#getting-started-with-codespace)
-  - [Creating your codespace](#creating-your-codespace)
+  - [Getting started with GitHub Codespaces](#getting-started-with-GitHub-Codespaces)
+  - [Creating your GitHub Codespaces](#creating-your-codespace)
   - [Using GitHub Codespaces in Visual Studio Code](#using-github-codespaces-in-visual-studio-code)
   - [Prerequisites](#prerequisites)
-  - [Opening a codespace in VS Code](#opening-a-codespace-in-vs-code)
+  - [Opening a GitHub Codespaces in VS Code](#opening-a-codespace-in-vs-code)
   - [Navigating to an existing codespace](#navigating-to-an-existing-codespace)
 ---
-## Getting started with codespace
+## Getting started with GitHub codespaces
 
 A codespace is a development environment that's hosted in the cloud. You can customize your project for GitHub Codespaces by committing configuration files to your repository, which creates a repeatable codespace configuration for all users of your project.
 
 ---
-## Creating your codespace
+## Creating your GitHub codespaces
 
-1. Click on    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/)
-1. New window will open where you can create a new Codespace
+1. Click on    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new/)
+1. New window will open where you can create a new codespace
 1. First you will need to select your repository you forked from the Information Assistant repo
-1. Next, by default you will be on **main** branch, you can switch to specific branch where you would like to create a Codespace.
-1. Then select options for **Region** and **Machine type**. The "Dev Container configuration" will be prepopulated and does not need to be changed.
+1. Next, by default you will be on **main** branch, you can switch to specific branch where you would like to create a Codespaces.
```

Comment:
when "codespace" is used not in conjunction with "GitHub Codespaces" it should not be capitalized.

Reply:
Sure. I will update accordingly